plank/cmd/main.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019-2021 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+package main
+
+import (
+	"github.com/urfave/cli"
```

Comment:
We're going to replace this with `cobra` still, right?

Reply:
yes, that was on my back burner, but now that most of the things are squared away i'll work on moving away from urfave cli

Reply:
this is now resolved as of the latest commit